GameData/ROCapsules/PartConfigs/CST/CSTCM.cfg

```diff
@@ -0,0 +1,392 @@
+		hibernationMultiplier = 0.625
+
+		RESOURCE
+		{
+			name = ElectricCharge
+			rate = 1.6
+		}
+	}
+	
+	MODULE
+	{
+		name = ModuleSAS
+		SASServiceLevel = 3
+	}
+	
+	MODULE
+	{
+		name = AdjustableCoMShifter
+		DescentModeCoM = 0.0, 0.0, 0.3
+	}
+	
+	MODULE
+	{
+		name = ModuleScienceExperiment
+		experimentID = crewReport
+		experimentActionName = Crew Report
+		resetActionName = Discard Crew Report
+		reviewActionName = Review Report
+		useStaging = False
+		useActionGroups = True
+		hideUIwhenUnavailable = True
+		rerunnable = True
+		xmitDataScalar = 1.0
+		usageReqMaskInternal = 5
+		usageReqMaskExternal = -1
+	}
+	
+	MODULE
+	{
+		name = ModuleScienceContainer
+		reviewActionName = Review Stored Data
+		storeActionName = Store Experiments
+		evaOnlyStorage = True
+		storageRange = 2.5
+	}
+	
+	MODULE
+	{
+		name = ModuleRCSFX
+		thrusterTransformName = RCS
+		thrusterPower = 0.445
+		resourceFlowMode = STAGE_PRIORITY_FLOW
+		runningEffectName = rcs
+		
+		PROPELLANT
+		{
+			name = Hydrazine
+			ratio = 1.0
+		}
+
+		PROPELLANT
+		{
+			name = Helium
+			ratio = 10.0
+			ignoreForIsp = True
+		}
+
+		atmosphereCurve
+		{
+			key = 0 220
+			key = 1 82
+		}
+	}
+	
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = ServiceModule
+		volume = 180
+		basemass = -1
+
+		//	Batteries 1.5 kWh.
+
+		TANK
+		{
+			name = ElectricCharge
+			amount = 5400
+			maxAmount = 5400
+		}
+
+		//	ACS propellant mass 150 Kg.
+
+		TANK
+		{
+			name = Hydrazine
+			amount = 150
+			maxAmount = 150
+		}
+
+		//	ACS pressurization gas mass ~0.27 Kg.
+
+		TANK
+		{
+			name = Helium
+			amount = 1500
+			maxAmount = 1500
+		}
+	}
+	
+	MODULE
+	{
+		name = FlagDecal
+		textureQuadName = flagTransform
+	}
+}
+
+//correct internals?
+@PART[ROC-CSTCM]:FOR[RealismOverhaul]
+{
+	@INTERNAL[CST100internal]
+	{
+		%scaleAll = 1.24, 1.24, 1.24
+
+		@MODULE[InternalSeat],*
+		{
+			%kerbalScale = 1.24, 1.24, 1.24
+			%kerbalOffset = 0.0, 0.0, 0.0
+			%kerbalEyeOffset = 0.0, 0.0, 0.0
+		}
+	}
+}
+
+//	==================================================
+//	CST-100 command module.
+
+//	Remote Tech compatibility.
+//	==================================================
+
+@PART[ROC-CSTCM]:FOR[RealismOverhaul]:NEEDS[RemoteTech]
+{
+	!MODULE[ModuleDataTransmitter],*{}
+
+	!MODULE[ModuleSPU*],*{}
+
+	!MODULE[ModuleRTAntenna*],*{}
+
+	@MODULE[ModuleCommand]
+	{
+		@RESOURCE[ElectricCharge]
+		{
+			@rate -= 0.025
+		}
+	}
+
+	MODULE
+	{
+		name = ModuleSPU
+	}
+
+	MODULE
+	{
+		name = ModuleRTAntenna
+		IsRTActive = True
+		Mode0OmniRange = 0
+		Mode1OmniRange = 800000
+		EnergyCost = 0.025
+		DeployFxModules = 0
+		ProgressFxModules = 1
+
+		TRANSMITTER
+		{
+			PacketInterval = 1.0
+			PacketSize = 1.024
+			PacketResourceCost = 0.01
+		}
+	}
+}
+
+//	==================================================
+//	CST-100 command module.
+
+//	Real Antennas compatibility.
+//	==================================================
+@PART[ROC-CSTCM]:BEFORE[RealAntennas]:NEEDS[RealAntennas]
```

Comment:
:BEFORE[x]:NEEDS[x] is redundant; BEFORE implies NEEDS. But it's common, so nbd.